From: Steven Robbins <smr@debian.org>
Date: Sun, 8 Apr 2018 23:20:35 -0500
Subject: Use GNU Install Dirs for multiarch.

---
 googlemock/CMakeLists.txt | 2 ++
 googletest/CMakeLists.txt | 2 ++
 2 files changed, 4 insertions(+)

diff --git a/googlemock/CMakeLists.txt b/googlemock/CMakeLists.txt
index beb259a..53eed13 100644
--- a/googlemock/CMakeLists.txt
+++ b/googlemock/CMakeLists.txt
@@ -5,6 +5,8 @@
 # ctest.  You can select which tests to run using 'ctest -R regex'.
 # For more options, run 'ctest --help'.
 
+include(GNUInstallDirs)
+
 # BUILD_SHARED_LIBS is a standard CMake variable, but we declare it here to
 # make it prominent in the GUI.
 option(BUILD_SHARED_LIBS "Build shared libraries (DLLs)." OFF)
diff --git a/googletest/CMakeLists.txt b/googletest/CMakeLists.txt
index 823c28c..355f427 100644
--- a/googletest/CMakeLists.txt
+++ b/googletest/CMakeLists.txt
@@ -5,6 +5,8 @@
 # ctest.  You can select which tests to run using 'ctest -R regex'.
 # For more options, run 'ctest --help'.
 
+include(GNUInstallDirs)
+
 # BUILD_SHARED_LIBS is a standard CMake variable, but we declare it here to
 # make it prominent in the GUI.
 option(BUILD_SHARED_LIBS "Build shared libraries (DLLs)." OFF)
